src/Image.cc

```diff
@@ -754,6 +765,21 @@ Image::decodeJPEGIntoSurface(jpeg_decompress_struct *args) {
 #if CAIRO_VERSION_MINOR >= 10
 
 /*
+ * Callback to recover from jpeg errors
+ */
+
+METHODDEF(void) canvas_jpeg_error_exit (j_common_ptr cinfo) {
+  // cinfo->err really points to a canvas_jpeg_error_mgr struct, so coerce pointer
+  canvas_jpeg_error_ptr cjerr = (canvas_jpeg_error_ptr) cinfo->err;
+
+  // Always display the message.
+  (*cinfo->err->output_message) (cinfo);
```

Comment:
I would like for this message to be returned in the `Error` object instead of always printed.